GBDaoGenerator/src/nodomain/freeyourgadget/gadgetbridge/daogen/GBDaoGenerator.java

```diff
@@ -265,6 +265,7 @@ private static Entity addNo1F1ActivitySample(Schema schema, Entity user, Entity
         activitySample.addIntProperty(SAMPLE_STEPS).notNull().codeBeforeGetterAndSetter(OVERRIDE);
         activitySample.addIntProperty(SAMPLE_RAW_KIND).notNull().codeBeforeGetterAndSetter(OVERRIDE);
         activitySample.addIntProperty(SAMPLE_RAW_INTENSITY).notNull().codeBeforeGetterAndSetter(OVERRIDE);
+        addHeartRateProperties(activitySample);
```

Comment:
That needs database migration code
see:

https://github.com/Freeyourgadget/Gadgetbridge/blob/master/app/src/main/java/nodomain/freeyourgadget/gadgetbridge/database/schema/GadgetbridgeUpdate_14.java

Reply:
Done.

```
09-10 15:05:23.960 25274-25274/nodomain.freeyourgadget.gadgetbridge I/nodomain.freeyourgadget.gadgetbridge.database.schema.SchemaMigration: ActivityDatabase: schema upgrade requested from 16 to 17
09-10 15:05:23.968 25274-25274/nodomain.freeyourgadget.gadgetbridge I/nodomain.freeyourgadget.gadgetbridge.database.schema.SchemaMigration: upgrading activity database to version 17
09-10 15:05:23.979 25274-25274/nodomain.freeyourgadget.gadgetbridge I/nodomain.freeyourgadget.gadgetbridge.database.schema.SchemaMigration: activity database is now at version 17
```

Looks like everything worked fine.